### PR TITLE
[15.0][ADD] stock_delivery_sale_section_and_note

### PR DESCRIPTION
**Pull request by StephaneMangin**
This module provides the propagation of sale order sections/notes lines to the delivery slip reports.

### CODE REVIEW

**Comment by leemannd** [comment]
Depends on -> https://github.com/OCA/sale-reporting/pull/235
Could you add the dependency (to be able to lauch the tests)?

**Comment by leemannd** [comment]
Hello @OCA/logistics-maintainers , your reviews are welcome

**Comment by rousseldenis** [comment]
Hi @StephaneMangin could you attend @leemannd comment about temporary reference ? 

https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

File: stock_delivery_sale_section_and_note/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+from odoo import fields, models
+
+from .stock_move_line import get_aggregated_properties
+
+
+class StockMove(models.Model):
+    _name = "stock.move"
+    _inherit = ["stock.move", "display.line.mixin"]
+
+    sequence = fields.Integer(related="sale_line_id.sequence", store=True)
```
Comment (rousseldenis) [inline stock_delivery_sale_section_and_note/models/stock_move.py:14]: My concern is about this line.

It is highly tight on sale line presence. If the delivery is generated from another flow, how does this behave ?

**Pull request closed without merging**